bindings/wasm/lib.rs

```diff
@@ -83,6 +98,18 @@ impl Statement {
         }
         array
     }
+
+    pub fn iterate(&self) -> JsValue {
+        let all = self.all();
```

Comment:
i'm a bit unsure about the utility of `iterate()` when we are materializing the results through `all()` anyway, wdyt?

Reply:
@jussisaurio it's API that better-sqlite has so we should have it too: https://github.com/WiseLibs/better-sqlite3/blob/master/docs/api.md. we can optimize `iterate()` to call step lazily?